src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.unit.cs

```diff
@@ -528,13 +528,13 @@ public void FileSystemWatcher_Path()
 
             // FSW starts with String.Empty and will ignore setting this if it is already set,
             // but if you set it after some other valid string has been set it will throw.
-            AssertExtensions.Throws<ArgumentException>(null, () => watcher.Path = String.Empty);
+            AssertExtensions.Throws<ArgumentException>("Path", () => watcher.Path = String.Empty);
```

Comment:
this is failing when run against desktop framework. you need to use this overload:
`public static void Throws<T>(string netCoreParamName, string netFxParamName, Action action)`